Ultimate vm

rtaylor4444 opened the pull request:
Fixes and adds new features for use in the Volcanic Mine mini game.
See the ReadMe for a more detailed description.

Nightfirecat commented:
[Is this config group the same as any other vm plugin?](https://github.com/rtaylor4444/UltimateVolcanicMine/blob/b0f8aa90cfaca1b998de1eb5b50ef7c36fad33d7/src/main/java/com/ultimatevm/UltimateVolcanicMineConfig.java#L6) Can you make it more specific to your plugin instead? (ex. `"ultimate-volcanic-mine"`)
LGTM otherwise

rtaylor4444 commented:
Thanks, I've fixed the issue that you found. There were also bugs that I found and fixed since I first made this PR so
those changes are now in as well.
